.github/workflows/sdk-build.yml

```diff
@@ -8,14 +8,18 @@ on: [push, pull_request]
 jobs:
   build:
     if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository) && !contains(github.event.head_commit.message, '[ci skip]')
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
```

Comment:
Please add `fail-fast: false` to the matrix definition so that if one build in the matrix fails, the others continue running (see [GitHub Actions docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures)).

Reply:
Thanks for the feedback 
let me know, now it is correct or not 😅